### ISSUE TITLE
0.17.3 not found in debian repository

### ISSUE THREAD

**Issue by hpurmann** [yarnpkg/yarn#1900]
**Do you want to request a *feature* or report a *bug*?**
Bug

**What is the current behavior?**
It still installs 0.16.1

**If the current behavior is a bug, please provide the steps to reproduce.**
Run `apt-get update && apt-get install -y yarn`

**What is the expected behavior?**
I want be able to upgrade manually by specifying the version of yarn in my `Dockerfile`:
```Dockerfile
FROM node:6.9.1
ARG YARN_VERSION=0.16.1-1

RUN apt-key adv --fetch-keys http://dl.yarnpkg.com/debian/pubkey.gpg \
    && echo "deb http://dl.yarnpkg.com/debian/ stable main" | tee /etc/apt/sources.list.d/yarn.list

RUN apt-get update && apt-get install -y \
    yarn=$YARN_VERSION \
    && rm -rf /var/lib/apt/lists/*
```

**Please mention your node.js, yarn and operating system version.**
Docker image extends from node 6.9.1

**Comment by wyze** [comment]
I would follow along #1859. I don't think there will be a release out to the official installation methods (e.g. Debian repository) until the checklist is finished.

**Comment by hpurmann** [comment]
Thanks for the quick answer, @wyze. Is there any general way to know when a new version has been officially shipped? Until now I just followed the GitHub Releases.

**Comment by tristanbes** [comment]
same question than @hpurmann I would except that github release would match what's available inside debian repository :(   // cc @wyze

**Comment by wyze** [comment]
Normally, once you see it on the GitHub releases, that is when it would be in all the official channels. It's just that there was a lot of breakage with 0.17.0, that they rolled back to 0.16.1 until we can fix the major regressions with 0.17.x.

**Comment by Daniel15** [comment]
Hello @hpurmann! @wyze is correct - 0.17.0 had quite a few regressions so we had to roll back. A fixed 0.17 release should be coming out today, at which point we'll update the Debian repo. Normally the Debian repo is in sync with the GitHub releases. Thanks for your patience 😄 

Alternatively, if you want to live on the edge, we have a 'nightly' Debian repo that contains the very latest code. I'd advise against doing that in production, but it's nice for testing. The source line for the testing repo is:
```
deb http://nightly.yarnpkg.com/debian/ nightly main
```
More info is available at https://yarnpkg.com/en/docs/nightly.
